base/util.jl

```diff
@@ -326,17 +326,18 @@ function with_output_color(f::Function, color::Union{Int, Symbol}, io::IO, args.
         if !iscolor
             print(io, str)
         else
+            bold && color == :bold && (color = :nothing)
+            enable_ansi  = get(text_colors, color, text_colors[:default]) *
+                               (bold ? text_colors[:bold] : "")
+            disable_ansi = get(disable_text_style, color, text_colors[:default]) *
+                               (bold ? disable_text_style[:bold] : "")
             lines = split(str, '\n')
             first = true
             for line in lines
```

Comment:
`for line in split(str, '\n')` would make it clearer that the `lines` variable is not used elsewhere.

Reply:
Done